### PR TITLE
test(icon): fix header typo in demo html page

### PR DESCRIPTION
**Pull request by ManahariDahal**
**Related Issue:** #2529 

## Summary
- fixed the pager header

### CODE REVIEW

**Comment by caripizza** [comment]
Thanks @ManahariDahal! Can you merge in master to this one? Hopefully those other file changes go away.

**Comment by ManahariDahal** [comment]
> Thanks @ManahariDahal! Can you merge in master to this one? Hopefully those other file changes go away.

I will merge it once it's approved :)

**Review by caripizza — APPROVED** [review]
Just needs latest master, thanks @ManahariDahal

**Pull request merged**
